requirements.txt

```diff
@@ -2,7 +2,7 @@
 # Each example subdir has its own `requirements.txt` with extra deps.
 # E.g. to install LLM folder deps, use `pip install -e .[llm]`
 
-mosaicml>=0.13.4,<0.14
+mosaicml>=0.13.4,<0.15
```

Comment:
should this be bumped to 0.14 min?

Reply:
llm is pinned to lower version still (not bothering to upgrade since we'll remove) so this is looser